### PR TITLE
test cases, some small fixes

### PR DESCRIPTION
**Pull request by offirgolan**
* Some simple integration tests for dummy app
* Some forgotten test cases for 1.0.2 bugfix release
* Forgotten link to messages in validation-result object
* upgrade ember-load-initializer
* upgrade ember-moment

### CODE REVIEW

**Comment by offirgolan** [comment]
@stefanpenner @blimmer any idea how to get these integration tests passing in 1.10 and 1.11?

Looks like there's an issue with setting up the initializer for `ember-cli-app-version`

**Comment by rwjblue** [comment]
Should be fixed by https://github.com/ember-cli/ember-load-initializers/pull/21 and released in ember-load-initializers@0.1.7.

**Comment by offirgolan** [comment]
@rwjblue coming through yet again! Thank you sir.

File: blueprints/ember-cp-validations/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   afterInstall: function() {
     return RSVP.all([
       this.addAddonToProject('ember-lodash', '0.0.5'),
-      this.addAddonToProject('ember-moment', '3.2.1')
+      this.addBowerPackageToProject('moment', '>= 2.8.0'),
```
Comment (blimmer) [inline blueprints/ember-cp-validations/index.js]: Why'd you go over to a global bower component?

Comment (offirgolan) [inline blueprints/ember-cp-validations/index.js]: This was causing some failures with ember ^2.0 but now that I look at it.... let me revert back and upgrade ember-moment to the latest version.

**Comment by offirgolan** [comment]
@blimmer ok to merge?

**Comment by blimmer** [comment]
:+1: sorry for the delay in reviewing.

**Comment by stefanpenner** [comment]
LGTM

**Pull request merged**